app.vue

```diff
@@ -54,30 +58,33 @@
       <div class="container mx-auto px-5 py-4 text-center">
         <p>
           Powered by
-          <a href="https://vercel.com/?utm_source=meshtastic&utm_campaign=oss" class="text-blue-500 hover:text-blue-700">▲ Vercel</a>
+          <a href="https://vercel.com/?utm_source=meshtastic&utm_campaign=oss">▲ Vercel</a>
           | Meshtastic® is a registered trademark of Meshtastic LLC. |
-          <a href="https://meshtastic.org/docs/legal" class="text-blue-500 hover:text-blue-700">Legal Information</a>.
+          <a href="https://meshtastic.org/docs/legal">Legal Information</a>.
         </p>
       </div>
     </footer>
   </div>
 </template>
 
 <script setup>
+import { ref, onMounted } from 'vue';
 import 'flowbite';
+import { initDropdowns, initModals, initTooltips } from 'flowbite';
+import { BoltIcon } from '@heroicons/vue/24/solid';
 
-import {
-  initDropdowns,
-  initModals,
-  initTooltips,
-} from 'flowbite';
+// WebSerial API support check
+const isWebSerialSupported = ref(false);
```

Comment:
I think instead of writing a method to populate the ref value onMounted, see if we can write it as a single computed: 

```
const isWebSerialSupported= computed(() => {
  return 'serial' in navigator;
};
```